tests/test_h1_client.c

```diff
@@ -29,7 +29,8 @@ static struct aws_http_message *s_new_default_get_request(struct aws_allocator *
     return request;
 }
 
-static void s_destroy_stream_on_complete(void *user_data) {
```

Comment:
Do we need regression test for the issue fixed?

Reply:
🙇‍♂️